ruby_event_store/spec/serialized_record_spec.rb

```diff
@@ -0,0 +1,19 @@
+module RubyEventStore
+  module Mappers
+    RSpec.describe SerializedRecord do
```

Comment:
I don't feel like we are testing anything here. Is this really necessary? If anything like `id, data, metadata, event_type` was not what was passed into it, then other tests would fail. The only thing which needs to be tested is `frozen?`.

It might make more sense if we introduce `Strings` verifications that I proposed.